woocommerce/migrator-cli
# Added command to import WooPayments data from Stripe

**Changes proposed**

This PR adds an import tool to pull the customer data from stripe. This is useful after doing a PAN import.
The script will get all customers from Stripe and save the customer_id to customers with the same email as the one in Stripe. 
It will also add the new payment methods to the customer accounts.


**Testing Instructions**
- Setup WooPayments
- Checkout to the server PR branch https://github.com/Automattic/woocommerce-payments-server/pull/4159
- Checkout to this PR branch
- Create a new customer
- Add a card via My account > Payment methods
- Place an order with that customer
- Now remove the `wp__wcpay_customer_id_test` and `wp__wcpay_customer_id_live` entries for the new customer from the `wp_usermeta` table
- Remove the payment tokens for that user from `wp_payment_tokens`
- Execute the import script `wp migrator import_stripe_data_into_woopayments`
- Go to My account > Payment methods make sure there is a payment method saved
- Place an order with that payment method to make sure it works
- Run the script again and make sure the payment method did not get duplicated
